### PR TITLE
br: add "tiup br debug" command

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by BornChanger — APPROVED** [review]

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
 * `tiup br backup`：用于备份 TiDB 集群的全量数据。
 * `tiup br log`：用于启动和管理日志备份任务。
 * `tiup br restore`：用于恢复备份数据到 TiDB 集群。
+* `tiup br debug`：用于解析备份元信息，检查备份数据等操作。
 
 `tiup br backup` 和 `tiup br restore` 还包含这些子命令：
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:37]:
```suggestion
`tiup br backup` 和 `tiup br restore` 包含以下子命令：
```

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
 
 * `full`：用于备份或恢复整个备份数据。
 * `db`：用于备份或恢复集群中的指定数据库。
 * `table`：用于备份或恢复集群指定数据库中的单张表。
 
+`tiup br debug` 包含这些子命令：
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:43]:
```suggestion
`tiup br debug` 包含以下子命令：
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
+
+* `checksum`：（隐藏参数）离线校验备份数据完整性，确保全部备份文件跟记录的 table checksum crc64 校验结果一致。
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:45]:
```suggestion
* `checksum`：（隐藏参数）离线校验备份数据完整性，确保全部备份文件跟记录的 [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) 计算的 CRC64 校验结果一致。
```

File: br/use-br-command-line-tool.md [hunk 4]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
+* `backupmeta`：检查备份数据文件是否有交集。正常情况下备份数据没有交集。
+* `decode`：将全量备份的元信息 backupmeta 文件解析成 json，并且支持通过 `--field` 参数解析特定的字段。
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:47]:
```suggestion
* `decode`：将全量备份的元信息 `backupmeta` 文件解析成 JSON 格式，并且支持通过 `--field` 参数解析特定的字段。
```

File: br/use-br-command-line-tool.md [hunk 5]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
+* `encode`：将全量备份的元信息 backupmeta.json 文件编码成恢复使用的 protobuf 格式。
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:48]:
```suggestion
* `encode`：将全量备份的元信息 `backupmeta.json` 文件编码成恢复数据时使用的 protobuf 格式。
```

File: br/use-br-command-line-tool.md [hunk 6]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
+* `reset-pd-config-as-default`：复原恢复过程中改变的 PD 设置（废弃）
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:49]: 要不要注明从哪个版本开始废弃的？
```suggestion
* `reset-pd-config-as-default`：复原数据恢复过程中改变的 PD 设置（已废弃）。
```

File: br/use-br-command-line-tool.md [hunk 7]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
 * `tiup br backup`：用于备份 TiDB 集群的全量数据。
 * `tiup br log`：用于启动和管理日志备份任务。
 * `tiup br restore`：用于恢复备份数据到 TiDB 集群。
+* `tiup br debug`：用于解析备份元信息，检查备份数据等操作。
 
-`tiup br backup` 和 `tiup br restore` 还包含这些子命令：
+`tiup br backup` 和 `tiup br restore` 还包含以下子命令：
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:37]:
```suggestion
`tiup br backup` 和 `tiup br restore` 包含以下子命令：
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: br/use-br-command-line-tool.md [hunk 8]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
 
+`tiup br debug` 包含以下子命令：
+
+* `checksum`：（隐藏参数）离线校验备份数据完整性，确保全部备份文件跟记录的 [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) 计算的 CRC64 校验结果一致。
+* `backupmeta`：检查备份数据文件是否有交集。正常情况下备份数据没有交集。
+* `decode`：将全量备份的元信息 `backupmeta` 文件解析成 JSON 格式，并且支持通过 `--field` 参数解析特定的字段。
+* `encode`：将全量备份的元信息 `backupmeta.json` 文件编码成恢复数据时使用的 protobuf 格式。
+* `reset-pd-config-as-default`：复原数据恢复过程中改变的 PD 设置（已废弃）。
+* `search-log-backup`：在日志备份数据中查找特定的 Key 信息。         
+
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:51]:
```suggestion
* `search-log-backup`：在日志备份数据中查找特定的 Key 信息。

```

File: br/use-br-command-line-tool.md [hunk 9]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
 * `full`：用于备份或恢复整个备份数据。
 * `db`：用于备份或恢复集群中的指定数据库。
 * `table`：用于备份或恢复集群指定数据库中的单张表。
 
+`tiup br debug` 包含以下子命令：
+
+* `checksum`：（隐藏参数）离线校验备份数据完整性，确保全部备份文件跟记录的 [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) 计算的 CRC64 校验结果一致。
+* `backupmeta`：检查备份数据文件是否有交集。正常情况下备份数据没有交集。
+* `decode`：将全量备份的元信息 `backupmeta` 文件解析成 JSON 格式，并且支持通过 `--field` 参数解析特定的字段。
+* `encode`：将全量备份的元信息 `backupmeta.json` 文件编码成恢复数据时使用的 protobuf 格式。
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:48]:
```suggestion
* `decode`：将全量备份的元信息文件 `backupmeta` 解析成 JSON 格式，并且支持通过 `--field` 参数解析特定的字段。
* `encode`：将全量备份的元信息文件 `backupmeta.json` 编码成恢复数据时使用的 protobuf 格式。
```

File: br/use-br-command-line-tool.md [hunk 10]
```diff
@@ -32,13 +32,23 @@ tiup br backup full --pd "${PD_IP}:2379" \
...
 * `db`：用于备份或恢复集群中的指定数据库。
 * `table`：用于备份或恢复集群指定数据库中的单张表。
 
+`tiup br debug` 包含以下子命令：
+
+* `checksum`：（隐藏参数）离线校验备份数据完整性，确保全部备份文件跟记录的 [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) 计算的 CRC64 校验结果一致。
+* `backupmeta`：检查备份数据文件是否有交集。正常情况下备份数据没有交集。
+* `decode`：将全量备份的元信息文件 `backupmeta` 解析成 JSON 格式，并且支持通过 `--field` 参数解析特定的字段。
+* `encode`：将全量备份的元信息文件 `backupmeta.json` 编码成恢复数据时使用的 protobuf 格式。
+* `reset-pd-config-as-default`：复原数据恢复过程中改变的 PD 设置（已废弃）。
```
Comment (lilin90) [inline br/use-br-command-line-tool.md:49]:
```suggestion
* `reset-pd-config-as-default`：（已废弃）将数据恢复过程中改变的 PD 设置复原为默认配置。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**